packages/salesforcedx-vscode-lwc/src/index.ts

```diff
@@ -107,7 +102,18 @@ export async function activate(context: ExtensionContext) {
   console.log('WorkspaceType detected: ' + workspaceType);
 
   // Start the LWC Language Server
-  startLWCLanguageServer(context);
+  const client = createLanguageClient(
+    context.asAbsolutePath(
```

Comment:
Have we tested this with the generated vsix ? It looks correct but we should confirm